Enterprise-CMCS/macpro-mdct-mfp
# MDCT-2737: Review and Submit Page

### Description

Adding a _very_ barebones version of the `ReviewSubmitPage` component and its related content.

**NOTE:** Sidebar integration isn't included in this PR since we don't have real report routes just yet.

### Related ticket(s)


MDCT-2737

---
### How to test

- Navigate to `/reviewSubmit` and see a page that says you've (somehow) submitted ✅ 

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
